### PR TITLE
fix: better handling of Ctrl-C and Ctrl-D in Readline

### PR DESCRIPTION
**Pull request by loicbourgois**

### CODE REVIEW

File: internal/interactive/prompt.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func Readline(config *ReadlineConfig) (string, error) {
 			case "Interrupt":
 				return "", &InterruptError{}
```
Comment (jerome-quere) [inline internal/interactive/prompt.go]: I think it would be better to return a CliError with an exit code here

File: internal/core/bootstrap.go [hunk 2]
```diff
@@ -70,6 +71,12 @@ func Bootstrap(config *BootstrapConfig) (exitCode int) {
 
 	rootCmd.SetArgs(config.Args[1:])
 	err = rootCmd.Execute()
+
+	switch err.(type) {
```
Comment (jerome-quere) [inline internal/core/bootstrap.go]: we should add support for an ExitCode field inside CLI error. and return 1 by default

Comment (loicbourgois) [reply internal/core/bootstrap.go]: where should we transform `interactive.InterruptError` into a `CliError` ?

File: internal/interactive/prompt.go (hunk 1 above)
Comment (loicbourgois) [reply internal/interactive/prompt.go]: are we sure we want to return a `CliError` from `interactive` ?

File: internal/core/bootstrap.go [hunk 3]
```diff
@@ -70,6 +71,13 @@ func Bootstrap(config *BootstrapConfig) (exitCode int) {
...
+	case *interactive.InterruptError:
+		return 130
+	}
+	fmt.Println("e", err)
```
Comment (jerome-quere) [inline internal/core/bootstrap.go]: shoud be removed

File: internal/core/bootstrap_test.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/args"
+	"github.com/scaleway/scaleway-cli/internal/interactive"
+)
+
+func getCommands() *Commands {
```
Comment (jerome-quere) [inline internal/core/bootstrap_test.go]: this "pollute" core test package let's write:
```
 func TestInterruptError(t *testing.T) {
  commands := NewCommands(&Command{...})
  t.Run(....)
}
```

**Pull request merged**